src/message_log.rs

```diff
@@ -127,11 +127,11 @@ impl PbftLog {
     ///  + A `PrePrepare` message matching the original message (in the current view)
     ///  + `2f + 1` matching `Prepare` messages from different nodes that match
     ///    `PrePrepare` message above (including its own)
-    pub fn prepared(&self, deser_msg: &PbftMessage, f: u64) -> Result<(), PbftError> {
-        if deser_msg.get_info().get_msg_type() != String::from(&PbftMessageType::Prepare) {
+    pub fn prepared(&self, pbft_message: &PbftMessage, f: u64) -> Result<(), PbftError> {
+        if pbft_message.get_info().get_msg_type() != String::from(&PbftMessageType::Prepare) {
             return Err(PbftError::NotReadyForMessage);
         }
-        let info = deser_msg.get_info();
+        let info = pbft_message.get_info();
```

Comment:
Should this go above the above line, since that has `pbft_message.get_info()` as well?